text/0000-t-test.md

```diff
@@ -1,71 +1,106 @@
+Currently, the overall automated testing experience spans multiple components owned by different
+teams across the Rust Project (e.g. T-cargo with `cargo test` being the primary
+touch point for most users, T-libs for libtest, T-rustdoc for doctests, etc.).
+This makes it more difficult to establish a vision and coordinate efforts to achieve that vision.
+However, there isn't any single team focused on the testing picture holistically.
+Simultaneously, there are a number of well known needs and feature requests in the space that do not have an explicit
+owner driving the efforts to completion.
 
-Before this, there was a growing interest in improving testing generally, like [better CI integration](https://internals.rust-lang.org/t/pre-rfc-implementing-test-binary-list-format-json-for-use-by-ide-test-explorers-runners/18308).
+For example, there's been some long standing requests to have additional test output
+formats, such as JSON ([#49359]) and JUnit, available on Stable Rust.
+While some of these are available as unstable features in Nightly Rust,
+in [Rust 1.70.0](https://blog.rust-lang.org/2023/06/01/Rust-1.70.0.html), a bug was fixed
+so unstable test features, like `cargo test -- --format json`, require using a nightly
+toolchain, like other unstable features. This caused breakage in certain editor/IDE and CI
+related tooling as they had been relying on the prior behavior to get test data
+programmatically, and little progress has been made in the last 5 years ([#49359]).
 
-One challenge with improving the situation is that a lot of concerns cross multiple teams, in particular cargo (for `cargo test`), libs (for libtest).  This makes it more difficult to establish a vision and coordinate efforts to achieve that vision.
+Furthermore, there's been a growing interest in improving testing generally, like [better CI integration][ci] as well as requests for things like better support for custom test harnesses and frameworks (.e.g [#2318]).
 
-There is also precedence for this today with t-devtools having more specific sub-teams like rustdoc and rustfmt.
+The new Testing team is intended to establish an overarching vision and provide focused attention on these areas. 
+
+[#49359]: https://github.com/rust-lang/rust/issues/49359
+[#50297]: https://github.com/rust-lang/rust/issues/50297
+[#2318]: https://github.com/rust-lang/rfcs/pull/2318
+[ci]: (https://internals.rust-lang.org/t/pre-rfc-implementing-test-binary-list-format-json-for-use-by-ide-test-explorers-runners/18308)
 
 ## Mission and responsibilities
 [mission]: #mission
 
-This team would primarily be focused on iterating on the test writing experience, `cargo test`, and enabling integration with external tools like CI or IDEs.
+This team would be primarily focused on iterating on the test writing and reporting experience, `cargo test`, and enabling integration points and features for external tools like CI or IDEs.
```

Comment:
What are your thoughts on "analysis" instead of reporting?

I chose "test writing" to focus on the higher level end-goal / experience. I think reporting is a an aspect of the intent of analyzing test state.

Reply:
Yeah that makes sense to me, and I'll change it later today. I think the reporting/analysis piece was already implicitly covered by the tie-ins with the external tools, but there's so many open threads around the reporting front that i thought it would be worth making explicit.

I like analysis better though :+1: